gridt/src/app/home/home.page.html

```diff
@@ -76,8 +59,12 @@ <h3>
             </ion-grid>
           </ion-card-content>
         </ion-card>
-    </ion-col>
-  </ion-row>
+      </ion-col>
+    </ion-row>
   </ion-grid>
 
+  <ng-template #Tutorial>
+    <app-tutorial></app-tutorial>
```

Comment:
I don't think that the 'app' in `app-tutorial` contributes much to the understanding of what this component does. I would suggest keeping it to simply tutorial.

Reply:
The app-'something' is the name of the component selector, so that this page appears here. For all pages, the selector starts with app, e.g. app-add-movement, or app-home. So I stuck to this convention also for the tutorial page.